core/src/test/java/org/bitcoinj/protocols/payments/PaymentSessionTest.java

```diff
@@ -60,7 +59,7 @@
     private ECKey serverKey;
     private Transaction tx;
     private TransactionOutput outputToMe;
-    private Coin coin = COIN;
```

Comment:
I think we should just inline this field. Otherwise it's just a "copied constant".

Reply:
Actually, I think we should rename it to `amount`. It's not a constant that specifies 1 coin, it's a constant that specifies a test amount that must match in the before and after state.

Reply:
I rebased, renamed `coin` to `amount`, and force-pushed. I'll mark this as resolved and await your review of the updated push.